R/declare_assignment.R

```diff
@@ -70,6 +71,13 @@ assignment_handler <-
                         !!cond_prob := obtain_condition_probabilities(!!!options, assignment = !!assn),
        ID_label = NA
       )
+      if(append_probabilities_matrix) {
+        options$N <- quo(nrow(data))
```

Comment:
When this loops back around to the second assignment variable, N is added to options, so `conduct_ra(N=N, N=nrow(data))` may break since N is NSE. Please either add a test to verify this works for multiple assn variables.